doc/api/dgram.md

```diff
@@ -344,13 +354,17 @@ added: v8.7.0
 
 * Returns: {number} the `SO_RCVBUF` socket receive buffer size in bytes.
 
+Throws [`ERR_SOCKET_BUFFER_SIZE`][] if called on an unbound socket.
```

Comment:
This also doesn't have a subject but we make lots of exceptions for `Throws` throughout the docs, I think, so I think this and the subsequent ones are OK.

Reply:
Added subject to all the sentences. Thanks for the review!